test/testcafe/spec/ChallengePIV_spec.js

```diff
@@ -59,7 +62,7 @@ test
     const pivPageObject = new PIVPageObject(t);
     await pivPageObject.navigateToPage();
     // wait for save to be triggered automatically
-    await t.wait(2000);
+    await t.wait(3000);
```

Comment:
For some reason it consistently failed using 2 seconds so I increased to 3 and it consistently passes.